modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -44,15 +44,31 @@
 
 require_once('include/ListView/ListViewData.php');
 
+include_once 'include/Exceptions/SuiteException.php';
+include_once 'modules/Emails/Folder.php';
+include_once 'modules/Emails/include/ListView/ListViewDataEmailsSearchOnCrm.php';
+include_once 'modules/Emails/include/ListView/ListViewDataEmailsSearchOnIMap.php';
+
+
+
 class ListViewDataEmails extends ListViewData
 {
+
+
+    /**
+     * enum('crm', 'imap')
+     *
+     * @var string
+     */
+    protected $searchType;
```

Comment:
We should implement an abstract class with constants. This simplifies the type in the php doc.